pkg/entrypoint/metricsbridge/telemetry.go

```diff
@@ -0,0 +1,57 @@
+package metricsbridge
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	metricsBridgeRegistry = prometheus.NewRegistry()
+
+	// TODO(charlesakalugwu): Add unit tests for the handling of these metrics once
+	//  the upstream library supports it
+	metricsBridgeErrorsCounter = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "metrics_bridge_errors_total",
+			Help: "Total number of errors.",
```

Comment:
non-blocking nitpick: make the ending period on all Help strings the same :smiley: (I'd vote for no ending period, not sure what's the "normal" here)

Reply:
Thanks @thekad :D The norm is actually for HELP strings to have a period at the end. I missed that on `metrics_bridge_processing_duration_seconds`. Thanks!